### PR TITLE
EZP-29650: Fix Selection field (Content Type fields) arrow color

### PR DESCRIPTION
**Pull request by tischsoic**
| Question      | Answer
| ------------- | ---
| Tickets       | https://jira.ez.no/browse/EZP-29650
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

Fixes 1. problem - pointer cursor.
2. problem was fixed in: https://github.com/ezsystems/ezplatform-admin-ui/pull/648 by the by of other task.
Previous PR to this task was closed: https://github.com/ezsystems/ezplatform-admin-ui/pull/644.
More info in JIRA.

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: src/bundle/Resources/public/scss/core/_custom.dropdown.scss [hunk 1]
```diff
@@ -3,7 +3,7 @@
         position: relative;
         color: $ez-color-base-dark;
         font-weight: 700;
-        cursor: pointer;
+        cursor: initial;
```
Comment (dew326) [inline src/bundle/Resources/public/scss/core/_custom.dropdown.scss]: Do we need `initial`? Cannot we just remove it?

Comment (sunpietro) [reply src/bundle/Resources/public/scss/core/_custom.dropdown.scss]: If you don't need it, then remove it.

Comment (tischsoic) [reply src/bundle/Resources/public/scss/core/_custom.dropdown.scss]: It overrides: `.ez-field-edit--ezselection .ez-data-source` which has `cursor: pointer;`.

Comment (sunpietro) [reply src/bundle/Resources/public/scss/core/_custom.dropdown.scss]: So, maybe you should fix it there, hm?

Comment (tischsoic) [reply src/bundle/Resources/public/scss/core/_custom.dropdown.scss]: I didn't wanted to break something.

Comment (sunpietro) [reply src/bundle/Resources/public/scss/core/_custom.dropdown.scss]: `ez-custom-dropdown` comes from `ezselection` fieldtype. You can remove it safely.

**Comment by tischsoic** [comment]
ping @sunpietro

**Pull request merged**